.circleci/config.yml

```diff
@@ -48,10 +48,6 @@ workflows:
           name: *name
           matrix:
             parameters: { solidus: ["current"], ruby: ["3.2"], db: ["mysql"] }
-      - run-specs:
-          name: *name
-          matrix:
-            parameters: { solidus: ["older"], ruby: ["3.1"], db: ["sqlite"] }
```

Comment:
Instead of removing it I think it's better to shift all the numbers, and test against 3.4, 3.3 and 3.2